Makefile

```diff
@@ -7,6 +7,9 @@ unit:
 integration:
 	node_modules/.bin/cucumber-js --tags $(INTEGRATIONS_TAGS) tests/cucumber/features --require-module ts-node/register --require tests/cucumber/steps/index.js
 
+display-all-js-steps:
+	tail -n +135 tests/cucumber/steps/steps.js | grep -v '^ *//' | awk "/(Given|Then|When)/,/',/" | grep -E "\'.+\'"  | sed "s/^[^']*'\([^']*\)'.*/\1/g"
```

Comment:
Generates input for Unused Steps Analysis script

Reply:
I'm a little out of the loop - is this script in the parent SDK testing repo?

Reply:
This make command is only here. However, it's output is used by the SDK testing repo's new [Cucumber Steps Analysis notebook](https://github.com/algorand/algorand-sdk-testing/blob/eaf31ff4188a6e2337435128a48db6f04498e5b0/utils/unused_steps_analysis.ipynb).